javascript/javascript3/week3/javaScript3-homework-week3/ScreenShot/app.js

```diff
@@ -0,0 +1,101 @@
+import env from "./env.json" assert {type:"json"}
+
+document.addEventListener("DOMContentLoaded", () => {
+  const screenshotForm = document.getElementById("screenshotForm");
+  const screenshotContainer = document.getElementById("screenshotContainer");
+  const savedScreenshotsContainer = document.getElementById("savedScreenshots");
+
+  screenshotForm.addEventListener("submit", async (event) => {
+    event.preventDefault();
+
+    const urlInput = document.getElementById("urlInput");
+    const url = urlInput.value;
+
+    // Generate the screenshot
+    await generateScreenshot(url);
+  });
+
+  async function generateScreenshot(url) {
+    const screenshotApiUrl = `https://website-screenshot6.p.rapidapi.com/screenshot?url=${encodeURIComponent(
+      url
+    )}&width=1920&height=1080&fullscreen=true`;
```

Comment:
To make it more responsive you can use the screen object instead of hardcoding the screen dimensions:
```javascript
const screenWidth = window.screen.width;
const screenHeight = window.screen.height;

console.log(`Screen width: ${screenWidth}`);
console.log(`Screen height: ${screenHeight}`);
```